test/unit_tests/conftest.py

```diff
@@ -235,8 +77,24 @@ def db_uri(worker_id):
     return str(url)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def init_db_obj(db_uri):
+    """
+    Intialise the database object, and register the Database class in the Wrapper.
+
+    Args:
+        db_uri: The database uri
+
+    Yields:
+        Nothing
+
+    """
+    db.update(Database(db_uri))
+    yield
+
+
```

Comment:
Why not move this into the database fixture itself?

Anyway it does not need autouse since its a dep from an autouse fixture